torchvision/utils.py

```diff
@@ -151,7 +152,7 @@ def draw_bounding_boxes(
     """
     Draws bounding boxes on given image.
     The values of the input image should be uint8 between 0 and 255.
-
+    Resulting Tensor should be saved as PNG image.
```

Comment:
Could you add a space in between doc sections? There is another PR that is trying to fix the styling on documentation. :)